# SPDX-License-Identifier: BSD-2-Clause
# Origin: EmberBSD contributors (AI-assisted).
# Rebuild direct consumers for the libplist 2.8 ABI.

--- a/sysutils/gvfs/Makefile
+++ b/sysutils/gvfs/Makefile
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.149 2026/09/02 19:04:29 wiz Exp $
 
 DISTNAME=	gvfs-1.60.0
-PKGREVISION=	5
+PKGREVISION=	6
 CATEGORIES?=	sysutils
 MASTER_SITES=	${MASTER_SITE_GNOME:=sources/gvfs/${PKGVERSION_NOREV:S/./ /g:[1..2]:ts.}/}
 EXTRACT_SUFX=	.tar.xz
--- a/comms/libusbmuxd/Makefile
+++ b/comms/libusbmuxd/Makefile
@@ -1,6 +1,7 @@
 # $NetBSD: Makefile,v 1.2 2026/05/02 05:49:35 vins Exp $
 
 DISTNAME=	libusbmuxd-2.1.1
+PKGREVISION=	1
 CATEGORIES=	comms
 MASTER_SITES=	${MASTER_SITE_GITHUB:=libimobiledevice/}
 GITHUB_RELEASE=	${PKGVERSION_NOREV}
--- a/comms/libimobiledevice/Makefile
+++ b/comms/libimobiledevice/Makefile
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.3 2026/05/14 16:40:32 ryoon Exp $
 
 DISTNAME=	libimobiledevice-1.4.0
-PKGREVISION=	2
+PKGREVISION=	3
 CATEGORIES=	comms
 MASTER_SITES=	${MASTER_SITE_GITHUB:=libimobiledevice/}
 GITHUB_RELEASE=	${PKGVERSION_NOREV}
--- a/multimedia/uxplay/Makefile
+++ b/multimedia/uxplay/Makefile
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.2 2026/09/02 19:03:50 wiz Exp $
 
 DISTNAME=	uxplay-1.73.3
-PKGREVISION=	1
+PKGREVISION=	2
 CATEGORIES=	multimedia
 MASTER_SITES=	${MASTER_SITE_GITHUB:=FDH2/}
 GITHUB_TAG=	v${PKGVERSION_NOREV}
--- a/audio/forked-daapd/Makefile
+++ b/audio/forked-daapd/Makefile
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.62 2026/09/02 19:00:47 wiz Exp $
 
 DISTNAME=	forked-daapd-27.4
-PKGREVISION=	54
+PKGREVISION=	55
 CATEGORIES=	audio
 MASTER_SITES=	${MASTER_SITE_GITHUB:=ejurgensen/}
 GITHUB_RELEASE=	${PKGVERSION_NOREV}
--- a/security/libtatsu/Makefile
+++ b/security/libtatsu/Makefile
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.2 2026/05/14 16:42:01 ryoon Exp $
 
 DISTNAME=	libtatsu-1.0.5
-PKGREVISION=	1
+PKGREVISION=	2
 CATEGORIES=	security
 MASTER_SITES=	${MASTER_SITE_GITHUB:=libimobiledevice/}
 GITHUB_RELEASE=	${PKGVERSION_NOREV}
--- a/audio/libgpod/Makefile
+++ b/audio/libgpod/Makefile
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.102 2026/09/02 19:00:54 wiz Exp $
 
 DISTNAME=	libgpod-0.8.3
-PKGREVISION=	27
+PKGREVISION=	28
 CATEGORIES=	audio
 MASTER_SITES=	${MASTER_SITE_SOURCEFORGE:=gtkpod/}
 EXTRACT_SUFX=	.tar.bz2
--- a/misc/libimobiledevice-glue/Makefile
+++ b/misc/libimobiledevice-glue/Makefile
@@ -1,6 +1,7 @@
 # $NetBSD: Makefile,v 1.1 2026/05/01 19:49:47 vins Exp $
 
 DISTNAME=	libimobiledevice-glue-1.3.2
+PKGREVISION=	1
 CATEGORIES=	misc
 MASTER_SITES=	${MASTER_SITE_GITHUB:=libimobiledevice/}
 GITHUB_RELEASE=	${PKGVERSION_NOREV}
